app/controllers/jobs_controller.rb

```diff
@@ -74,7 +74,11 @@ def set_job
   def find_free_worker
     current_user
       .workers
-      .joins("left outer join jobs on jobs.worker_id = workers.id")
+      .active
+      .joins(
+          "left outer join jobs on jobs.worker_id = workers.id " +
```

Comment:
Do you need to write `"left outer join jobs ..."`, or can't you just write `"left join jobs ..."`?

Also, could you capitalize the SQL keywords, please?

Reply:
Ah yeah just left join works! And will do

Reply:
The reason I mentioned that is I found that out when studying for the DB midterm 😂 😂

Reply:
Truuue. Also I just added a commit from my phone on the Greyhound back to Waterloo lol